vault/activity_log_test.go

```diff
@@ -170,6 +173,8 @@ func checkExpectedEntitiesInMap(t *testing.T, a *ActivityLog, entityIDs []string
 	}
 }
 
+// TestActivityLog_UniqueEntities calls AddEntityToFragment 4 times with 2 different clients, then verifies that there
+// are only 2 clients in the fragment and that they have the earlier timestamps
```

Comment:
This is super nitpicky, so please feel free to ignore: the [official godocs documentation](https://go.dev/doc/comment) calls out that these should be complete sentences (implying that they should end in a period). A few of the comments look like they adhere, but it would be good to be consistent.